### PR TITLE
[AC][SUPP-510] enforce library upgrades

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -86,6 +81,22 @@ lazy val wartRemoverSettings =
     wartremoverExcluded ++= (baseDirectory.value ** "*.sc").get
   )
 
+lazy val sbtUpdatesSettings = Seq(
```
Comment (barnesjake) [inline build.sbt:84]: It's not one for this pr, but just a thought, should we look to separate out all various tool settings (wart remover, sbtupdates etc) into their own objects within the project directory? I feel like the build sbt is getting quite large and we could trim it down a lil bit

Comment (AndyHWChung) [reply build.sbt:84]: I may as well do it while I'm here - happy to do it 👍

**Review by barnesjake — DISMISSED** [review]

File: build.sbt (hunk 1 above)
Comment (barnesjake) [reply build.sbt:84]: Ok, I guess same for the other prs too?

Comment (AndyHWChung) [reply build.sbt:84]: yeah, I'll go through them now quickly

Comment (AndyHWChung) [reply build.sbt:84]: there's a ticket to go to 2.13 - I was gonna leave updating scala when we come round to that

Comment (barnesjake) [reply build.sbt:84]: Fair enough

Comment (AndyHWChung) [reply build.sbt]: just pushed up an update [here](https://github.com/hmrc/essttp-backend/pull/67/commits/840d6ad540109d7678b494d86d3b1387d1b7369d)

**Review by barnesjake — APPROVED** [review]

**Pull request merged**